src/Components/Auth/ForgotPassword/index.jsx

```diff
@@ -0,0 +1,77 @@
+import { Link, useHistory } from 'react-router-dom/cjs/react-router-dom.min';
+import { useForm } from 'react-hook-form';
+import { useDispatch } from 'react-redux';
+import styles from './forgotPassword.module.css';
+
+import { getAuth, sendPasswordResetEmail } from 'firebase/auth';
+import { handleDisplayToast, setContentToast } from 'Redux/Shared/ResponseToast/actions';
+
+import { joiResolver } from '@hookform/resolvers/joi';
+import resetPasswordSchema from 'Validations/forgotPassword';
+
+import Button from 'Components/Shared/Button';
+import { Input } from 'Components/Shared/Inputs';
+
+const ForgotPassword = () => {
+  const history = useHistory();
+  const dispatch = useDispatch();
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors }
+  } = useForm({
+    mode: 'onChange',
+    resolver: joiResolver(resetPasswordSchema),
+    defaultValues: {
+      email: ''
+    }
+  });
+
+  const handleSendEmail = (data) => {
+    const auth = getAuth();
+    sendPasswordResetEmail(auth, data.email, { url: `${process.env.REACT_APP_URL}/auth/login` })
+      .then(() => {
+        history.push('/auth/login');
+        dispatch(setContentToast({ message: 'Email with reset link sent', state: 'success' }));
+        dispatch(handleDisplayToast(true));
+      })
+      .catch(() => {
+        dispatch(setContentToast({ message: 'Email is not registed', state: 'fail' }));
```

Comment:
you have a typo here. It´s registered

Reply:
did you add the following code?

Add the following environment variable in the .env file:
REACT_APP_URL=http://localhost:3000/

Reply:
yes!